scss/_pagination.scss

```diff
@@ -182,6 +178,10 @@
     margin-left: var(--#{$prefix}pagination-margin-x-first-last);
   }
 
+  &:not(:first-child):not(:last-child) {
+    max-width: var(--#{$prefix}pagination-item-max-width);
```

Comment:
Since all the styles seems to be applied on `.page-link`, I think we should do so for this rule, meaning change `width` or `max-width`. (`$pagination-item-max-width: 10px` or the width of the icon).